diff --git a/includes/redhen_donation.forms.inc b/includes/redhen_donation.forms.inc
index 2ef831a..d178adf 100644
--- a/includes/redhen_donation.forms.inc
+++ b/includes/redhen_donation.forms.inc
@@ -630,7 +630,7 @@ function redhen_donation_form($form, &$form_state, RedhenDonation $donation, $us
     $product = current($products);
     $form['product'] = array(
       '#type' => 'hidden',
-      '#value' => $product->product_id,
+      '#value' => $product->sku,
     );
   }
   if (!$form_state['use_cart']) {
